sample-apps/SampleAppSpec.md

```diff
@@ -134,15 +134,17 @@ Asynchronous Gauge
 
 **Common Attributes for Random based metrics**
 
-These are Key Value pairs to be added on metrics.
+These are Key Value pairs to be added on metrics and will be tested for.
```

Comment:
```suggestion
These Key Value pairs MUST be present on metrics.
```
This conveys the same message in a more succint form. Must implies a requirement which means it should be tested for.

Reply:
I agree and will be changing to this message.